docs/site/decorators/Decorators_authenticate.md

```diff
@@ -40,5 +40,8 @@ export class WhoAmIController {
 }
 ```
 
+{% include note.html content="If only <b>some</b> of the controller methods are decorated with the <b>@authenticate</b> decorator, then the injection decorator for CURRENT_USER in the controller's constructor must be specified as <b>@inject(AuthenticationBindings.CURRENT_USER, {optional:true})</b> to avoid a binding error when an unauthenticated endpoint is accessed. Alternatively, do not inject CURRENT_USER in the controller <b>constructor</b>, but in the controller <b>methods</b> which are actually decorated with the <b>@authenticate</b> decorator. See [Method Injection](../Dependency-injection.md#method-injection), [Constructor Injection](../Dependency-injection.md#constructor-injection) and [Optional Dependencies](../Dependency-injection.md#optional-dependencies) for details.
+" %}
```

Comment:
I don't know if you need the double quotation here for the markdown file

Reply:
@agnes512 

Hi Agnes,

I followed the LoopBack documentation concerning documentation : tips, alerts, notes etc
https://loopback.io/doc/en/contrib/alerts.html

![image](https://user-images.githubusercontent.com/6864736/59794858-e6bf4800-92a7-11e9-816b-66023745ab08.png)

They use `"`.

But yes in javascript, `'` can be used in place of `"` in most situations.

Reply:
👍

Reply:
I think the double quotation is necessary, since `content="` begins with one.